lib/features/public_asset/data/network/public_asset_api.dart

```diff
@@ -134,4 +150,29 @@ class PublicAssetApi {
       throw const CannotUpdatePublicAssetException();
     }
   }
+
+  Future<void> partialUpdatePublicAssets(
+    Session session,
+    AccountId accountId,
+    {required Map<PublicAssetId, UpdatingIdentityIds> mapPublicAssetIdToUpdatingIdentityIds}
```

Comment:
can you test the case with multiple Entries in this map?

Reply:
Test updated for multiple entries with different update types